.circleci/config.yml

```diff
@@ -407,6 +397,18 @@ workflows:
               only:
                 - develop
     jobs:
-      - deb-tests
-      - translation-tests
-      - fetch-tor-debs
+      - deb-tests:
```

Comment:
@eloquence I am not sure if these changes are valid as these jobs only run on develop branch weekly, and `deb-tests` and `translation-tests` wouldn't run by these filters. But I made these changes as earlier while using the `match-ci-branch.sh` script, these filters used to apply anyway, and I was not sure if we wanted a change in that behavior or not.

Reply:
I agree that the `branches` specifications can be removed for jobs under the `weekly` workflow too.  Now that [CircleCI has deprecated `branches` specifications at the job level](https://circleci.com/docs/2.0/configuration-reference/#branches-deprecated), these would only be relevant if these jobs were added to the `securedrop_ci` or some other per-branch workflow.  The `weekly` configuration will be much simpler without them!